model/patch/patch.go

```diff
@@ -1060,10 +1061,12 @@ func GetCollectiveStatus(statuses []string) string {
 			hasFailure = true
 		case evergreen.PatchSucceeded:
 			hasSuccess = true
+		case evergreen.PatchAborted:
+			hasAborted = true
 		}
 	}
 
-	if !(hasCreated || hasFailure || hasSuccess) {
+	if !(hasCreated || hasFailure || hasSuccess || hasAborted) {
```

Comment:
I refactored it so that there is always statuses given (because we now handle aborted in the same place as the other statuses). I think that no statuses given would indicate some sort of error case because there should always be at least one status.